deb-get

```diff
@@ -2453,7 +2462,7 @@ case "${ACTION}" in
         ls -lh "${CACHE_DIR}/";;
     clean)
         elevate_privs
-        ${ELEVATE} rm -v "${CACHE_DIR}"/*.deb;;
+        ${ELEVATE} rm -v "${CACHE_DIR}"/*.deb
```

Comment:
Please refrain from changing things outside of the scope of your pull request. It helps keep things organized.